WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewController.swift

```diff
@@ -1,5 +1,11 @@
 import SwiftUI
 
+extension NSNotification.Name {
+    /// Posted whenever the hub menu view did appear.
+    ///
+    public static let setUpTapToPayViewDidAppear = Foundation.Notification.Name(rawValue: "com.woocommerce.ios.setUpTapToPayViewDidAppear")
```

Comment:
That's convenient! 💯

Reply:
Yep. Notifications need to be used with care, but I think it's appropriate here.